### ISSUE TITLE
Pop-Os boot black screen

### ISSUE THREAD

**Issue by dylanpjx** [pop-os/pop#626]
**Distribution (run `cat /etc/os-release`):**
 NAME="Pop!_OS"
VERSION="19.04"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.04"
VERSION_ID="19.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=disco
UBUNTU_CODENAME=disco
Nvidia Version

**Issue/Bug Description:**
I'm a linux beginner so I don't exactly know what is happening but PopOS always boots on a black screen for me. I've checked the other black screen issues and it seems to be different from mine. 
I always get a totally black screen with absolutely nothing on it. I then try to hit alt-f2 to reach command line so I can at least "startX" but the command line always seems to be busy. The cursor for the command line appears and blinks and I cannot type anything into the command line. I've tried using normal console termination like Ctrl-C or Ctrl-Z but the command line doesnt respond at all. The only way I manage to boot my pc up is to plug in my thumbdrive with the fresh install boot image into my pc and somehow PopOS will boot up correctly.

**Comment by jblood91** [comment]
I'm having the same issue with the AMD/Intel version. Try to boot, screen goes black and my monitor says no signal. The only way I can get to the desktop is through recovery mode. No idea where to go from there as I'm still new to Linux as well. Restarting from recovery mode still leads to a black screen during boot, so it doesn't really help.

**Comment by dylanpjx** [comment]
It's not really a fix but I think I found out the issue. It boots into a black screen when the terminal is still trying to complete a task and you just shut down the pc. After booting back popOs using the iso image on my thumbdrive, i noticed there were installation processes running for fonts. When I closed those installation processes, popOs booted properly without the help of the iso. I'm gonna leave this issue as open since idk whether the devs can make steps to ensure the terminal is not in use before shut down.

Check out this [site](url) for more information on how to kill running processes!
Also, in case you get into the same situation, you can press Alt+F2 to access the terminal and at least try to troubleshoot or launch the desktop environment with it. @jblood91

**Comment by jblood91** [comment]
Thanks for the info! I'll be researching how to kill those processes and see where it gets me. @dylanpjx

**Comment by hegemanjr** [comment]
This is happening to me too on my HP Spectre.

**Comment by officialcjunior** [comment]
Hello, I experienced the same issue, too. Pop_OS 20.04 with black screen, no output, and can't even make it to the Login Screen.

I solved this and got back to my Pop, by going into the recovery mode and running `fsck` on all my partitions. There was a problem in my `/dev/sda3` and `fsck` fixed it for me.

`sudo fsck -f /dev/sda3`

A reboot later, I was back on my desktop.

Hope it helps!

**Comment by kharevkk** [comment]
> Hello, I experienced the same issue, too. Pop_OS 20.04 with black screen, no output, and can't even make it to the Login Screen.
> 
> I solved this and got back to my Pop, by going into the recovery mode and running `fsck` on all my partitions. There was a problem in my `/dev/sda3` and `fsck` fixed it for me.
> 
> `sudo fsck -f /dev/sda3`
> 
> A reboot later, I was back on my desktop.
> 
> Hope it helps!

How did you get to the recovery mode? I have been trying by using alt+f2 and all I see is black screen with a cursor where I can't type.

**Comment by officialcjunior** [comment]
> > Hello, I experienced the same issue, too. Pop_OS 20.04 with black screen, no output, and can't even make it to the Login Screen.
> > I solved this and got back to my Pop, by going into the recovery mode and running `fsck` on all my partitions. There was a problem in my `/dev/sda3` and `fsck` fixed it for me.
> > `sudo fsck -f /dev/sda3`
> > A reboot later, I was back on my desktop.
> > Hope it helps!
> 
> How did you get to the recovery mode? I have been trying by using alt+f2 and all I see is black screen with a cursor where I can't type.

Hold space bar while booting.

**Comment by kharevkk** [comment]
> > > Hello, I experienced the same issue, too. Pop_OS 20.04 with black screen, no output, and can't even make it to the Login Screen.
> > > I solved this and got back to my Pop, by going into the recovery mode and running `fsck` on all my partitions. There was a problem in my `/dev/sda3` and `fsck` fixed it for me.
> > > `sudo fsck -f /dev/sda3`
> > > A reboot later, I was back on my desktop.
> > > Hope it helps!
> > 
> > 
> > How did you get to the recovery mode? I have been trying by using alt+f2 and all I see is black screen with a cursor where I can't type.
> 
> Hold space bar while booting.
Appreciate your response.
Only space bar ? Cuz its not working..i am stuck on blank screen.

**Comment by officialcjunior** [comment]
> > > > Hello, I experienced the same issue, too. Pop_OS 20.04 with black screen, no output, and can't even make it to the Login Screen.
> > > > I solved this and got back to my Pop, by going into the recovery mode and running `fsck` on all my partitions. There was a problem in my `/dev/sda3` and `fsck` fixed it for me.
> > > > `sudo fsck -f /dev/sda3`
> > > > A reboot later, I was back on my desktop.
> > > > Hope it helps!
> > > 
> > > 
> > > How did you get to the recovery mode? I have been trying by using alt+f2 and all I see is black screen with a cursor where I can't type.
> > 
> > Hold space bar while booting.
> Appreciate your response.
> Only space bar ? Cuz its not working..i am stuck on blank screen. 

Yes, space bar, then you'll get a menu like shown in this article.
https://support.system76.com/articles/pop-recovery/#:~:text=To%20boot%20into%20recovery%20mode,OS%20Recovery.

It might ask you to select the language and all, when you enter the recovery mood. Ignore that, press Super+T, and try running the fsck command.

I really hope that you'll get back to your Pop.

**Comment by kharevkk** [comment]
Yeah I checked the pop os community link and yeah you were right. But fortunately something unusual happened and I am back in the system. Thanks !

**Comment by kharevkk** [comment]
> > > > > Hello, I experienced the same issue, too. Pop_OS 20.04 with black screen, no output, and can't even make it to the Login Screen.
> > > > > I solved this and got back to my Pop, by going into the recovery mode and running `fsck` on all my partitions. There was a problem in my `/dev/sda3` and `fsck` fixed it for me.
> > > > > `sudo fsck -f /dev/sda3`
> > > > > A reboot later, I was back on my desktop.
> > > > > Hope it helps!
> > > > 
> > > > 
> > > > How did you get to the recovery mode? I have been trying by using alt+f2 and all I see is black screen with a cursor where I can't type.
> > > 
> > > 
> > > Hold space bar while booting.
> > > Appreciate your response.
> > > Only space bar ? Cuz its not working..i am stuck on blank screen.
> 
> Yes, space bar, then you'll get a menu like shown in this article.
> https://support.system76.com/articles/pop-recovery/#:~:text=To%20boot%20into%20recovery%20mode,OS%20Recovery.
> 
> It might ask you to select the language and all, when you enter the recovery mood. Ignore that, press Super+T, and try running the fsck command.
> 
> I really hope that you'll get back to your Pop.

Hi, I had posted same issue separately and someone told me if I am dual booting recovery mode isn't available. 
However, now i think the brave browser is causing problem. Yesterday it was working fine. I tired to open brave and it was stuck and kept loading on home page  i Uninstalled it and installed it again to resolve the issue it didn't work and then finally uninstalled it again. 

Today I installed again and still the browser wasn't working and when I restarted my system, I am in the same situation again. 

Any idea what to do on such case.

**Comment by edips** [comment]
I sometimes get the same issue on 20.04 with Asus, Nvidia/Intel. I sometimes get black screen after entering encryption password. When I shut it down and reboot, I am back to the desktop. I use Pop_OS for almost 2 years and I get this problem for 2 months. I reset Gnome, Pop desktop environments. I changed desktop environments like Cinnamon, LXDE, KDE.. I got the same issue. it isn't about desktop environment. Could it be because of video card drivers?

**Comment by ajay767** [comment]
@edips  i am also getting the same fault. whenever i start my laptop with pop os it goes into black screen, i can't even open tty. please help.

**Comment by edips** [comment]
@ajay767 It doesn't happen to me for every start. Yesterday it was with black screen, then I powered off my laptop by pressing power key for a while. When I reopened it, it worked. Sometimes moving the mouse helps. I don't know why.

**Comment by CertifiedRascal** [comment]
I just experienced this issue, and what fixed it for me was unplugging all external monitors, usb devices, and basically everything but the charging cable (computer is a Thinkpad W541). Without checking each cable one by one, I'm not sure which cable was what was locking everything up, but my guess would be it was the display cable. I also had a black screen with a blinking terminal underscore cursor, but you couldn't type anything. Hopefully this helps someone who comes upon this issue!

**Comment by Anas1921** [comment]
Hi i am new to linux , i tried to install pop os on my old laptop ,after installation completed the screen got blank  after waiting for an hour i turned off the laptop and turned on manually still it shows nothing and doesn't even show bios setup.my laptop is msi cr400

**Comment by kylemccurley** [comment]
Same issue here. This has completely killed my Thelio Major system. Just a black screen and no boot

**Comment by Bebotron** [comment]
Any news on this from official maintainers? I just installed 21.04 a handful of days ago, and it just started happening today. No idea how to fix other than reinstalling -- recovery mode doesn't thanks to legacy mode (https://github.com/system76/docs/issues/468#issuecomment-697733516)

**Comment by kiberptah** [comment]
reinstalled pop os 21.04 Nvidia twice, have this issue after first reboot... Black screen, caps lock doesnt work, space bar doesn't open recovery, just a brick machine state.

**Comment by idoodler** [comment]
I have the same issue with my issue. I can go to the "Boot Menu" to switch between Pop!_OS (current kernal), Pop!_OS (legacy kernal), Recovery Mode, Windows Boot Manager or Firmare Settings.

Only Windows Boot Manager and Firmware Settings work.

I tried booting into a Live Disk and run `fsck` as recogmended, but without any luck. This is the second time I experience this issue within 3 Days..

After experience it the first time I simply reinstalled everything and I was extreamly carefull when rebooting my system, like waiting until all processes finished, always use the reboot button of the gnome Desktop.

But I guess something with the partitions is messed up, the PC needs a long time to show any dissplay output, so I guess it is trying out different partitions according to the boot order.

**Comment by fierg** [comment]
> I just experienced this issue, and what fixed it for me was unplugging all external monitors, usb devices, and basically everything but the charging cable (computer is a Thinkpad W541). Without checking each cable one by one, I'm not sure which cable was what was locking everything up, but my guess would be it was the display cable. I also had a black screen with a blinking terminal underscore cursor, but you couldn't type anything. Hopefully this helps someone who comes upon this issue!

Removing all external monitors and only adding one solved the issue for me as well. this is realted to the nvidia driver

**Comment by behdadmansouri** [comment]
Mine **worked** with the "old kernel" option at boot menu
The issue was probably shutting down by force during an important kernel update
I booted the "old kernel" and uninstalled the newest one, [with the help of this reddit post](https://www.reddit.com/r/pop_os/comments/rbfpap/two_questions_ultra_simple_instructions_for/)

**Comment by Armfoot** [comment]
I have an **AMD Radeon graphics** card and have been using the same Pop!_OS (popos) for a few years without major issues.

Last week (3rd week of February, 2024) popos v22.04 system updates through the Pop!_Shop caused this **black screen issue where only the mouse pointer was seen and froze** after a few seconds. It was not possible to interact with popos (attempted all keys combos) and the old/new kernel startup menu (F5) would lead to the same frozen state, the console/command line would also not show up.

So using a USB flashed with popos at startup:
1. ignored the installation window
2. opened the _Files_ app and clicked on the disk partition with the OS (mounting it it) and went to `/etc/systemd/system/`
3. opened the _Terminal_ app with a right click on this folder (context menu)
4. renamed the link to prevent the graphical interface to be booted: `sudo mv display-manager.service display-manager.service.OLD` (**systemctl** command in this [reddit post](https://www.reddit.com/r/pop_os/comments/rlk2tq/how_to_boot_in_command_line_mode/?rdt=58849) removes this link, but you want to keep it for later though)
5. restarted without the USB plugged-in and was able to login with my user through console
6. added `blacklist radeon` on a new line of `sudo nano /etc/modprobe.d/blacklist.conf`, saving the file
7. ran `sudo update-initramfs -c -k all` this took almost an hour to complete (several warnings show up) but the process did finish
8. `sudo mv display-manager.service.OLD display-manager.service` this initiated the graphical interface upon the next step's restart
9. `sudo shutdown -r now`

This solution was based on System76 article: https://support.system76.com/articles/login-loop-pop/#blacklist-radeon-driver

> If you are using an AMD graphics card, the radeon driver might be causing issues as it does not support newer versions of Ubuntu. The amdgpu driver that comes with the kernel does however.

The **graphical login screen now shows up** and I'm able to use apps as usual, but I no longer am able to use an external monitor (doesn't even detect it) or adjust my laptop's monitor brightness.

I updated all packages and then reversed the above process (removed radeon from the blacklist), but the dreaded black screen showed up again (lost another couple of hours).
So I'm considering backing up everything and doing a clean install just to check if the drivers get fixed.

Wrote this for those pulling hairs off and trying not to smash the PC... Hope it helps!

**Comment by Armfoot** [comment]
Un [update on my previous answer][1]:

1. after a clean popos v22.04 install, the external monitor and the laptop's brightness keys were working properly 
2. once I used Pop!_Shop for the default OS updates, the black screen and I had to use the [same USB pen reboot to fix the boot][1]
3. attempted several solutions to install and revert Radeon drivers (even with proprietary drivers packages available on their site), none worked
4. downloaded Lubuntu v23.10 (Mantic Minotaur), did a clean install and updated packages/system
5. what a a surprise, the external monitor works as expected and the laptop's monitor brightness can be adjusted with a GUI that comes with the system
 
## Summary

Even though the amdgpu driver allows to use popos (thanks devs that wrote that [black screen issues article](https://support.system76.com/articles/login-loop-pop/#blacklist-radeon-driver)), the setbacks are too great to continue using it, so I'm now on Lubuntu and I'm likely not going back.

[1]: https://github.com/pop-os/pop/issues/626#issuecomment-1970893376

**Comment by davidwindell** [comment]
Same issue here just now as @Armfoot, just did an update and couldn't boot. Only option was to blacklist radeon but then only a single display works. This sucks.
